packages/@okta/vuepress-site/docs/reference/api/xaas/index.md

```diff
@@ -0,0 +1,493 @@
+
+#### Use example
+
+The following request cancels an Import Session with an `id` value of `${sessionId}`:
+
+##### Request
+
+```bash
+curl -X DELETE
+https://${yourOktaDomain}/api/v1/identity-sources/${identitySourceId}/sessions/${sessionId}
+-H 'accept: application/json'
+-H 'authorization: SSWS ${apiToken}'
+-H 'cache-control: no-cache'
+-H 'content-type: application/json'
+```
+
+##### Response
+
+```http
+HTTP/1.1 204 No Content
+Content-Type: application/json
+```
+
+## Anything-as-a-Source API objects
+
+### Import Session object
+
+#### Import Session properties
+
+| Property    | Type           | Description   |
+| ----------- | -------------- | ------------- |
+| `id` | String | The unique identifier for the Import Session |
+| `identitySourceId` | String | The unique identifier obtained from creating a Customer Identity Source integration in Okta |
+| `status` | Enum: `CREATED`, `TRIGGERED`, `COMPLETED`, `CLOSED`, `EXPIRED` | The current status of the Import Session:<br><ul><li>CREATED: New Import Session that hasn't been processed. You can load bulk user data in this stage.</li><li>TRIGGERED: Okta is processing the import data in this session. You can't load bulk user data in this stage.</li><li>COMPLETED: The bulk user data was processed and imported into Okta.</li><li>CLOSED: The Import Session was canceled and isn't available for further activity.<li>EXPIRED: This Import Session had the `CREATED` status and timed-out after 24 hours of inactivity.</li></ul>|
+
+<!--Murty Devarakonda (Oct 20): ERROR status isn't documented because it will be removed.-->
+
+#### Import Session example
+
+```json
+{
+  "id": "{sessionId}",
+  "identitySourceId": "{identitySourceId}",
+  "status": "CREATED",
+  "importType": "INCREMENTAL"
+}
+```
+
+### External Profile object
+
+#### External Profile properties
+
+| Property           | Type                           | Description               |
+| ------------------ | ------------------------------ | ------------------------------ |
+| `externalId`        | String                 | The unique identifier for the user in the external HR source |
+| `profile`          | [Profile object](/docs/reference/api/users/#profile-object) | Contains a set of external user attributes and their values that are mapped to Okta standard and custom profile properties. See [Profile object](/docs/reference/api/users/#profile-object) and Declaration of a Custom Identity Source Schema in [Using anything as a source](https://help.okta.com/okta_help.htm?type=oie&id=ext-anything-as-a-source)  |
+
+#### External Profile object example
+
+```json
+{
+  "externalId": "{extUserId}",
+  "profile": {
+    "userName": "isaac.i.brock@example.com",
+    "firstName": "Isaac",
+    "lastName": "Brock",
+    "email": "isaac.i.brock@example.com",
+    "secondEmail": "ibrock.test@example.com",
+    "mobilePhone": "555-123-4567",
+    "homeAddress": "Kirkland, WA"
+  }
+}
+```
+
+#### Possible common errors
```

Comment:
Should this section be further up the hierarchy? Maybe this:
```suggestion
## Possible common errors
```

Reply:
Yes, let's move this section up the hierarchy. I looked at the other APIs and H2 was reserved for operation and objects, so I converted this section to H3 instead.